content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -499,7 +499,8 @@ stages:
 <admon type="tip">
 
 Both individual foreach stages (`train@1`) and groups of foreach stages
-(`train`) may be used in commands that accept stage targets.
+(`train`) may be used in commands that accept stage targets such as
+`dvc repro` and `dvc stage list`.
 
 </admon>
```

Comment:
Wait though... Shouldn't this note be much higher up after the example with foreach stages named `train`? Currently it's under a seemingly unrelated code bloc @dberenbaum 👍🏼 

<img width="1143" alt="image" src="https://user-images.githubusercontent.com/1477535/193990504-ce092c7e-1058-4696-b9d0-54b642b87a47.png">

> See https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#foreach-stages

Reply:
Or maybe just one block up before the `mystages` code block and modify the text to include example of syntax for dictionary:

Both individual foreach stages (`build@uk` / `train@1`) and groups of foreach stages (`build` / `train`)

Reply:
Wherever you think is best @jorgeorpinel

Reply:
Updated and moved.